torntools/scripts/content/loader/ttAttackLoader.js

```diff
@@ -5,10 +5,17 @@ requireDatabase().then(() => {
 });
 
 function displayWarning() {
-	if (userdata.energy.current > userdata.energy.maximum && userdata.chain.current > 10) {
-		let rawHTML = `<div class='tt-overlay-div'><span class='tt-overlay-text'>Warning! You have stacked energy. Beware!</span><button class='tt-overlay-button'>OK</button></div>`;
+	if (userdata.energy.current > userdata.energy.maximum && userdata.chain.current < 10 && new Date(settings.disable_attack_stacked_warning_until) - new Date() <= 0) {
+		let rawHTML = `<div class='tt-overlay-div'><span class='tt-overlay-text'>Warning! You have stacked energy. Beware!</span><button class="tt-silver-button tt-ok-button">OK</button><button class="tt-silver-button tt-dnd-button">I'm Chaining/Warring</button></div>`;
 		doc.find("a[href='#skip-to-content']").insertAdjacentHTML("afterEnd", rawHTML);
-		doc.find("button.tt-overlay-button").addEventListener("click", (event) => (event.target.parentElement.style.display = "none"));
+		doc.find("button.tt-silver-button.tt-ok-button").addEventListener("click", (event) => event.target.parentElement.style.display = "none");
+		doc.find("button.tt-silver-button.tt-dnd-button").addEventListener("click", (event) => {
+			event.target.parentElement.style.display = "none";
+			ttStorage.get("settings", (settings) => {
```

Comment:
It would be better to do a `ttStorage.change({ settings: { andMoreDeeperInTheSettings:true } });`

Reply:
I actually don't know that function and I saw the code to do that in ttForums.js where you change the hide_posts. I implemented that. Will change it.

Reply:
Done.